packages/shared/src/features/onboarding/shared/utils.ts

```diff
@@ -18,3 +18,15 @@ export const sanitizeMessage = (message: string): string => {
     ALLOWED_ATTR: [],
   });
 };
+
+const brokenWebviewPatterns = [
+  /FBAN|FBAV/i, // Facebook App
+  /Messenger/i, // Facebook Messenger
+  /LinkedIn/i, // LinkedIn
```

Comment:
@idoshamun this is so far what I've found to be broken.

Reply:
Can we somehow enable it for when window.open isn't available? Worst case this list is great. We gonna run mostly fb campaigns

Reply:
IIRC, the window.open is available. It just doesn't work (blocked). Otherwise, the app would've thrown a runtime error. Let me double-check.

Reply:
Tried it on the latest commit, and yeah window.open is available but just doesn't work. Hence we can't use it to check whether we should do redirect.

Reply:
let's stick to this list. as long as we have facebook covered we're good to good 👌

Reply:
Agreed :shipit: